### PR TITLE
Add mocked response for GIBill Statement of Benefits

### PR DESCRIPTION
**Pull request by AdamKing0126**
This PR is to add mocked responses for Post-911 Gi Bill Statement of benefits at 
`/education/gi-bill/post-9-11/ch-33-benefit/`

Use this mock like this:
`yarn mock-api --responses src/applications/post-911-gib-status/mocks/server.js`

This will mock out the responses to the statement of benefits tool.  Included is only a 200 response.  Inspect the [API Docs](https://developer.va.gov/explore/api/education-benefits/docs?version=current) for other responses

### CODE REVIEW

File: src/applications/post-911-gib-status/mocks/server.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+/* eslint-disable camelcase */
+const responses = {
```
Comment (nicksayre) [inline src/applications/post-911-gib-status/mocks/server.js:2]:
```suggestion
const responses = {
  'GET /v0/backend_statuses/gibs': {
    data: {
      id: 'string',
      type: 'string',
      attributes: {
        isAvailable: true,
        name: 'gibs',
      },
    },
  },
```

**Review by nicksayre — CHANGES_REQUESTED** [review]
Added one quick code suggestion.

**Comment by AdamKing0126** [comment]
> Added one quick code suggestion.

Thanks Nick!

**Review by nicksayre — APPROVED** [review]

File: src/applications/post-911-gib-status/mocks/server.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (AdamKing0126) [reply src/applications/post-911-gib-status/mocks/server.js:1]: Sorry, gotta keep it!

**Review by CBonade — APPROVED** [review]

**Pull request merged**